source/usage-examples/find.txt

```diff
@@ -0,0 +1,143 @@
+=======================
+Find Multiple Documents
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+`collection.find() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#find>`_
+lets you query for multiple documents in MongoDB. You can pass ``find``
+an (optional) query document and an options object. Use the options
+object to specify a sort order and a projection, both of which
+are optional. Consult the `collection.find() reference documentation <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#find>`_
+to learn about other things you can accomplish with options and
+``find``.
+
+Query Document
+~~~~~~~~~~~~~~
+
+You can use the query document to filter the collection down from all
+documents to a specific subset. Consider a collection containing the
+following documents describing fruit:
+
+.. code-block:: javascript
+
+   { "_id": 1, "name": "apples", "qty": 5, "rating": 3 }
+   { "_id": 2, "name": "bananas", "qty": 7, "rating": 1 }
+   { "_id": 3, "name": "oranges", "qty": 6, "rating": 2 }
+   { "_id": 4, "name": "avocados", "qty": 3, "rating": 5 }
+
+You could pass the following query to the ``find`` method to search for
+only fruit documents listing a quantity of greater than 5:
+
+.. code-block:: javascript
+
+   const query = { qty: { $gt : 5 } };
+
+``find`` returns the following documents for such a query:
+
+.. code-block:: javascript
+
+   { "_id": 2, "name": "bananas", "qty": 7, "rating": 1 }
+   { "_id": 3, "name": "oranges", "qty": 6, "rating": 2 }
+
+Sort Order
+~~~~~~~~~~
+
+You can use the sort order to change the order in which documents are
+returned by ``find``. You could pass the following sort order to
+``find`` to ensure that fruit documents with higher ratings are returned
+before fruit documents with lower ratings:
+
+.. code-block:: javascript
+
+   const sort = { rating: -1 }
+
+In this case, the number ``-1`` tells ``find`` to sort the fruit in
+descending order by rating, so the highest rated fruit comes first and
+the lowest rated fruit comes last. ``find`` returns the following
+documents when this sort is used with no query:
+
+.. code-block:: javascript
+
+   { "_id": 4, "name": "avocados", "qty": 3, "rating": 5 }
+   { "_id": 1, "name": "apples", "qty": 5, "rating": 3 }
+   { "_id": 3, "name": "oranges", "qty": 6, "rating": 2 }
+   { "_id": 2, "name": "bananas", "qty": 7, "rating": 1 }
+
+
+If we rewrote this sort to use a value of
```

Comment:
Active voice and simplification. As an example `Rewriting this sort to use a value of **1** instead of **-1** will sort in ascending order.`